staging/src/k8s.io/client-go/tools/clientcmd/loader.go

```diff
@@ -37,6 +37,7 @@ import (
 	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
 	clientcmdlatest "k8s.io/client-go/tools/clientcmd/api/latest"
 	"k8s.io/client-go/util/homedir"
+	"k8s.io/kubernetes/pkg/util/slice"
```

Comment:
@sttts client-go can't import k8s.io/kubernetes, can it?

Reply:
It can't. The func is simple enough that we can put it here into the file at the end.